pkg/apis/cni/v1beta1/cni_grpc.pb.go

```diff
@@ -0,0 +1,197 @@
+// Copyright 2019 Antrea Authors
```

Comment:
Do you know why the year info is still 2019? is this generated manually?

Reply:
My guess is that protoc automatically uses the information from the .proto source file, but I didn't look into it.